qemu/tests/numa_consistency.py

```diff
@@ -45,9 +45,17 @@ def get_vcpu_used_node(numa_node_info, vcpu_thread):
     params['vcpu_cores'] = 1
     params['vcpu_threads'] = 1
     params['vcpu_sockets'] = params['smp']
+    params['vcpu_maxcpus'] = params['smp']
     params['guest_numa_nodes'] = ""
+    params['mem_devs'] = ""
+    params['backend_mem'] = "memory-backend-ram"
+    params['use_mem'] = "no"
+    params['size_mem'] = "1024M"
```

Comment:
How about move L50~L53 to cfg ?

Reply:
Is it strange to define an empty memdev with size and other options?

Reply:
En, make sense, let's keep the current status.